sites/main-site/src/content/blog/2024/helpdesk.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Announcing: Weekly Helpdesk'
+subtitle: 'A weekly drop-in sessions to connect and work together.'
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Announcing: Weekly Helpdesk'
+    start: 2024-10-28T16:14:00+01:00
+    end: 2024-10-28T16:14:00+01:00
+pubDate: 2024-10-28T16:14:00+01:00
```

Comment:
Publishing now? Or start of week / after my nf-core talk at the summit?

After my talk at summit times:
```suggestion
    start: 2024-10-31T16:18:00+01:00
    end: 2024-10-31T16:18:00+01:00
pubDate: 2024-10-31T16:18:00+01:00
```